### PR TITLE
feat: add branding prop for customise QR branding

### PR DESCRIPTION
**Pull request by jellydn**
_Issue #, if available:_

_Description of changes:_
 - Add a new prop for customizing the issuer. As my screenshot, it's showing `AWSCognito`.
image
 - By this PR, we could change to `YourCompanyName` as an issuer.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-ui-components/src/components/amplify-totp-setup/amplify-totp-setup.tsx [hunk 1]
```diff
@@ -27,6 +27,8 @@ export class AmplifyTOTPSetup {
   @Prop() handleAuthStateChange: AuthStateHandler = dispatchAuthStateChangeEvent;
   /** Used for header text in totp setup component */
   @Prop() headerText: string = I18n.get(Translations.TOTP_HEADER_TEXT);
+   /** Used for customise the branding in qr code image */
+  @Prop() branding: string = 'AWSCognito';
```
Comment (ashika01) [inline packages/amplify-ui-components/src/components/amplify-totp-setup/amplify-totp-setup.tsx:31]: Just a quick note - we add all constants to `constants.ts` and reference it elsewhere - https://github.com/aws-amplify/amplify-js/blob/main/packages/amplify-ui-components/src/common/constants.ts

If there is a possibility to have translated string here, then probably add it to [Translations.ts](https://github.com/aws-amplify/amplify-js/blob/main/packages/amplify-ui-components/src/common/Translations.ts)

Comment (jellydn) [reply packages/amplify-ui-components/src/components/amplify-totp-setup/amplify-totp-setup.tsx]: That's a good idea. I have just implemented in the way you mentioned. Thanks.

File: packages/amplify-ui-components/src/components/amplify-totp-setup/amplify-totp-setup.tsx [hunk 2]
```diff
@@ -26,7 +26,9 @@ export class AmplifyTOTPSetup {
   /** Auth state change handler for this component */
   @Prop() handleAuthStateChange: AuthStateHandler = dispatchAuthStateChangeEvent;
   /** Used for header text in totp setup component */
-  @Prop() headerText: string = Translations.TOTP_HEADER_TEXT;
+  @Prop() headerText: string = I18n.get(Translations.TOTP_HEADER_TEXT);
+   /** Used for customise the branding in qr code image */
+  @Prop() branding: string = I18n.get(Translations.TOTP_BRANDING);
```
Comment (jordanranz) [inline packages/amplify-ui-components/src/components/amplify-totp-setup/amplify-totp-setup.tsx:31]: Could we use the prop name "issuer" here? (per [Google Authenticator documentation](https://github.com/google/google-authenticator/wiki/Key-Uri-Format#issuer))

File: packages/amplify-ui-components/src/components/amplify-totp-setup/amplify-totp-setup.tsx [hunk 3]
```diff
@@ -26,7 +26,9 @@ export class AmplifyTOTPSetup {
   /** Auth state change handler for this component */
   @Prop() handleAuthStateChange: AuthStateHandler = dispatchAuthStateChangeEvent;
   /** Used for header text in totp setup component */
-  @Prop() headerText: string = Translations.TOTP_HEADER_TEXT;
+  @Prop() headerText: string = I18n.get(Translations.TOTP_HEADER_TEXT);
+   /** Used for customise the branding in qr code image */
```
Comment (jordanranz) [inline packages/amplify-ui-components/src/components/amplify-totp-setup/amplify-totp-setup.tsx:30]:
```suggestion
   /** Used for customizing the issuer string in the qr code image */
```

File: packages/amplify-ui-components/src/components/amplify-totp-setup/amplify-totp-setup.tsx (hunk 2 above)
Comment (jordanranz) [reply packages/amplify-ui-components/src/components/amplify-totp-setup/amplify-totp-setup.tsx:31]: `const issuer` used later in the component could be changed to `encodedIssuer`

Comment (jellydn) [reply packages/amplify-ui-components/src/components/amplify-totp-setup/amplify-totp-setup.tsx:31]: Good idea. Thanks.

**Pull request merged**